htdocs/compta/tva/index.php

```diff
@@ -88,7 +90,7 @@ function pt ($db, $sql, $date)
             print '<td class="nowrap">'.$obj->dm."</td>\n";
             $total = $total + $obj->mm;
 
-            print '<td class="nowrap" align="right">'.price($obj->mm)."</td><td >&nbsp;</td>\n";
+            print '<td class="nowrap" align="right">'.price(price2num($obj->mm,1))."</td><td >&nbsp;</td>\n";
```

Comment:
We should not make any rounding here. 
If data is a total amount, rounding must be 'MT'  if a unit price, it must be 'MU' 
But here there is no reason to make rounding, rounding must be done before inserting data into table.